ignite/distributed/comp_models/native.py

```diff
@@ -241,15 +251,15 @@ def _setup_env_in_slurm(self) -> None:
             os.environ["LOCAL_RANK"] = os.environ["SLURM_LOCALID"]
             os.environ["WORLD_SIZE"] = os.environ["SLURM_NTASKS"]
             # port should be the same over all process
-            slurm_port = os.environ["SLURM_JOB_ID"]
+            slurm_port = os.environ["SLURM_JOBID"]
```

Comment:
In the docs for `srun`  they say:
> SLURM_JOB_ID (and SLURM_JOBID for backwards compatibility)
    Job id of the executing job.

Shouldn't we use `SLURM_JOB_ID` instead ?

https://slurm.schedmd.com/srun.html

Reply:
You are right !! The deprecated variable is `SLURM_JOBID`